tests/test_benchmark_spec.py

```diff
@@ -12,7 +14,11 @@ def _generate_test_path(p):
 
 
 def test_get_input_file_size():
-    assert _minimal_spec.get_input_file_size() == 161
+    size = _minimal_spec.get_input_file_size()
+    if platform.system() == 'Windows':
+        assert size == 167
+    else:
+        assert size == 161
```

Comment:
Can you please add some commentary to explain this?

Reply:
Added, the sample file [cat.ion](https://github.com/amazon-ion/ion-python/blob/master/tests/benchmark_sample_data/sample_spec/cat.ion) has different file size according to the operating systems.